rust/value.rs

```diff
@@ -76,8 +148,37 @@ impl Spanned for Literal {
 impl Pretty for Literal {}
 
 impl fmt::Display for Literal {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str(&self.inner)
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        self.inner.fmt(f)
```

Comment:
This is ambiguous between `Display` and `Debug` (probably where the import came from), should be `fmt::Display::fmt(self.inner, f)`